src/components/dialogs/dialogAlert/dialogAlertFooter/dialogAlertFooter.tsx

```diff
@@ -0,0 +1,57 @@
+import { Action, Cancel } from '@radix-ui/react-alert-dialog';
+import classNames from 'classnames';
+import type React from 'react';
+import { useContext } from 'react';
+import { Button, type IButtonProps } from '../../../button';
+import { DialogAlertContext } from '../dialogAlertRoot';
+
+export interface IDialogAlertFooterProps {
+    /**
+     * Alert dialog primary action button
+     */
+    actionButton: IButtonProps & { label: string };
```

Comment:
Should we omit the `variant` & `size` properties? In any case, I would suggest to create a new interface like:
```
IDialogAlertFooterAction extends IButtonProps {
  /**
   * [...]
   */
  label: string;

```